libvast/src/system/partition.cpp

```diff
@@ -905,8 +905,8 @@ partition_actor::behavior_type passive_partition(
       return rp;
     },
     [self](atom::erase) -> caf::result<atom::done> {
-      VAST_WARN("erasing partition {} from path", self->state.id,
-                self->state.path);
+      VAST_TRACE("erasing partition {} from path", self->state.id,
+                 self->state.path);
```

Comment:
```suggestion
      VAST_DEBUG("erasing partition {} from path", self->state.id,
                  self->state.path);
```

Even too much in DEBUG?

Reply:
That should also be OK, since we already have a couple of `DEBUG` messages that are sent once in a partition's lifetime.